android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java

```diff
@@ -68,6 +68,11 @@ public void onNewIntent(Intent intent) {
             bundle.putBoolean("foreground", false);
             intent.putExtra("notification", bundle);
             notifyNotification(bundle);
+        }else if ( intent.getExtras() != null ) {
+            Bundle bundle = intent.getExtras();
+            bundle.putBoolean("foreground", false);
+            intent.putExtra("notification", bundle);
+            notifyNotification(bundle);
         }
     }
```

Comment:
I'd be happy to merge this, but I can't just test it.  What do I need to do in order to invoke the new code?